ts/utils/walletv3.ts

```diff
@@ -0,0 +1,47 @@
+import { pipe } from "fp-ts/lib/function";
```

Comment:
Why did you place this file here and not in `ts/features/walletV3` folder?

Reply:
I moved it into `ts/features/walletV3/onboarding/utils` folder since in this file there are only onboarding-specific utils.

Addressed in [2d50633](https://github.com/pagopa/io-app/pull/4810/commits/2d5063304f55de1653688acedb5a5b1d3beeabc3)